pkg/virt-controller/watch/migration.go

```diff
@@ -451,10 +461,8 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 		vmi.Status.MigrationState.MigrationUID == migration.UID &&
 		vmi.Status.MigrationState.Failed {
 
-		migrationCopy.Status.Phase = virtv1.MigrationFailed
-		c.recorder.Eventf(migration, k8sv1.EventTypeWarning, FailedMigrationReason, "Source node reported migration failed")
-		handleFailedMigrationMetrics(migration, vmi, pod)
-		log.Log.Object(migration).Errorf("VMI %s/%s reported migration failed.", vmi.Namespace, vmi.Name)
+		c.failMigration(false, migrationCopy, vmi, pod)
+
```

Comment:
how does the failMigration() functino improve this here?

Reply:
removed :+1: